app/locales/en.json

```diff
@@ -2,14 +2,125 @@
   "actions": {
     "docs": "Docs",
     "examples": "Examples",
-    "github": "GitHub",
-    "getInTouch": "Get in touch"
+    "github": "Github",
+    "getInTouch": "Get in touch",
+    "search": "Search"
   },
   "404": {
     "title": "Page not found"
   },
   "footer": {
     "title": "Made with ❤️ across the globe",
     "version": "Api version"
+  },
+  "main": {
+    "preview": {
+      "title": "GammaCV is a WebGL accelerated Computer Vision library for modern web applications.",
+      "text": "We created GammaCV to make it easy to integrate Computer Vision in modern web applications. GammaCV was built with a similar architecture to TensorFlow, in particular, it also uses a data flow paradigm to create and run graphs on GPU, this enables the robust image processing and feature extraction capability of the library."
+    },
+    "facts": {
+      "compact": {
+        "title": "Compact",
+        "text": "Weighing in at only 87.5k uncompressed and 32.5k compressed GammaCV is small enough to be used in your Progressive Web Application without weighing it down."
+      },
+      "fast": {
+        "title": "Fast",
+        "text": "Many computer vision libraries make the typical laptop sound like it is about to take off like a jet plane. Not GammaCV, our use of WebGL enables fast frame rates without consuming all of the users CPU resources."
+      },
+      "cross_platform": {
+        "title": "Cross Platform",
+        "text": "We did the work so you don\"t have to. All of the capabilities offered by GammaCV have been tested to work reliably in all modern browsers on desktop, mobiles and tablets."
+      }
+    }
+  },
+  "docs": {
+    "groups": {
```

Comment:
The locale file has a lot of duplication from `docs.(groups|operations)` and `examples.(groups|operations)`. We should remove it to simplify localization using.